### DISCUSSION TITLE
With termux, jupyter, scipy, PYroMat is a blessing on Android

### DISCUSSION THREAD

**Discussion by defencedog** [chmarti1/PYroMat#50] [Show and tell]
Chemical engineer from Pakistan.

Now this really is a blessing for me on Android! Most of libraries are dependent upon `SCIPY` which I fail to install on _termux_ but `PYroMat` is OK

**Comment by chmarti1** [comment]
Thanks for taking the time to let us know how you're using PYroMat.  Reducing the number of dependencies was always a priority, so it's great to hear that it's paying off.  Happy computing!
